ibis/backends/dask/tests/test_cast.py

```diff
@@ -53,9 +52,14 @@ def test_cast_string(t, df, from_, to, expected):
 @pytest.mark.parametrize(
     ("to", "expected"),
     [
-        ("string", "object"),
+        pytest.param(
+            "string",
```

Comment:
I think this raise will also apply to `"string[pyarrow]"`.

Reply:
I would assume so too.